README.md

```diff
@@ -0,0 +1,33 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
```

Comment:
- [ ] Kindly update the readme file, this is a Todo list project and not an awesome book